lua_examples/telnet/telnet.lua

```diff
@@ -134,6 +134,7 @@ local function telnet_listener(socket)
   socket:on("disconnection", disconnect)
   socket:on("sent",          sendLine)
   node.output(queueLine, 0)
+  print(("Welcome to NodeMCU world (%d mem free, %s)"):format(node.heap(), wifi.sta.getip()))
```

Comment:
I think this should be left where it is, below; the IP address is important to print out before the first connection, right?

Reply:
I like to see this message when I connect to the telnet server. This is the way it works in the simple telnet example.
At the initial location I would expect message like:
```Lua
print(("Telnet server started (%d mem free, %s)"):format(node.heap(), wifi.sta.getip()))
```

Reply:
OK, so add something like that.

Reply:
Done